runtime/syntax/pike.vim

```diff
@@ -1,142 +1,385 @@
 " Vim syntax file
 " Language:	Pike
-" Maintainer:	Francesco Chemolli <kinkie@kame.usr.dsi.unimi.it>
```

Comment:
Could you change this line to "Previous Maintainer:" instead of removing?

Reply:
Fixed.